src/Language/DifferentialDatalog/Compile.hs

```diff
@@ -610,7 +612,7 @@ compileLib d specname modules rs_code = (typeLib, mainLib)
     -- Add 'DDValConvert' impls to 'types' crate.
     -- Since multiple DDlog types can map to the same Rust type, we convert
     -- types to Rust and fold over the resulting set.
-    typeLib = S.foldl (\libs t -> M.adjust ($+$ "::differential_datalog::decl_ddval_convert!{" <> pp t <> "}") mainModuleName libs) typeLibAllFuncs
+    typeLib = S.foldl (\libs _ -> M.adjust ($+$ "") mainModuleName libs) typeLibAllFuncs
                       $ S.map (render . mkType d True) $ S.filter (typeNeedsDDValConvert d) $ cTypes cstate
```

Comment:
I couldn't figure out how to totally remove the `($+$ "")`, so help there would be appreciated